infrastructure/indexer-provision.sh

```diff
@@ -4,11 +4,17 @@ set -x # Show commands
 set -eu # Errors/undefined vars are fatal
 set -o pipefail # Check all commands in a pipeline
 
+# We currently try to keep the version of clang we use matching the one that
+# will be used by the Firefox build process.  If you have a "mach bootstrap"ped
+# system then you can see the current version locally via
+# ` ~/.mozbuild/clang/bin/clang --version`
```

Comment:
So for some reason that I can't figure out, the backticks in this comment are causing the provisioning step to just silently fail. I run the trigger-provision.py script which seems to send the correct stuff to AWS, but then when AWS initializes the instance it never seems to get or execute the provisioning script and I don't know why.

At any rate, mind force-pushing a version that doesn't have these backticks?

Reply:
(I changed this to double-quotes.)